client/my-sites/stats/stats-date-picker/index.jsx

```diff
@@ -126,10 +124,12 @@ class StatsDatePicker extends Component {
 		const isToday = today.isSame( date, 'day' );
 		return (
 			<span>
-				{ translate( 'Last update: %(time)s', {
+				{ translate( '{{b}}Last update: %(time)s{{/b}} (Refreshes every 30 minutes)', {
```

Comment:
Would "Updates" be more user friendly and accurate than "refreshes"?

Reply:
Yep. On it.

Reply:
Changed.